Gems/AtomLyIntegration/CommonFeatures/Code/Source/Material/EditorMaterialComponent.cpp

```diff
@@ -388,7 +272,8 @@ namespace AZ
                 if (slot.m_id.IsLodAndSlotId())
                 {
                     // Resize the containers to fit all elements
-                    m_materialSlotsByLod.resize(AZ::GetMax<size_t>(m_materialSlotsByLod.size(), aznumeric_cast<size_t>(slot.m_id.m_lodIndex + 1)));
+                    m_materialSlotsByLod.resize(
+                        AZ::GetMax<size_t>(m_materialSlotsByLod.size(), aznumeric_cast<size_t>(slot.m_id.m_lodIndex + 1)));
                     m_materialSlotsByLod[slot.m_id.m_lodIndex].push_back(slot);
```

Comment:
should this be `emplace_back`

Reply:
It could be if there was an optimization with a move but the container has been reduced in size substantially so there may not be much to gain